src/Nancy/NancyModule.cs

```diff
@@ -263,80 +380,35 @@ public virtual ModelValidationResult ModelValidationResult
         }
 
         /// <summary>
-        /// Helper class for configuring a route handler in a module.
+        /// Declares a route for the module
         /// </summary>
-        public class RouteBuilder : IHideObjectMembers
+        /// <typeparam name="T"></typeparam>
+        /// <param name="name">Name of the route</param>
+        /// <param name="method">The HTTP method that the route will response to</param>
+        /// <param name="path">The path that the route will respond to</param>
+        /// <param name="action">Action that will be invoked when the route it hit</param>
+        /// <param name="condition">A condition to determine if the route can be hit</param>
+        protected void AddRoute<T>(string name, string method, string path, Func<NancyContext, bool> condition, Func<dynamic, CancellationToken, T> action)

```

Comment:
They weren't really swapped around, but moved out of necessity. The `AddRoute` parameter order is left unchanged while the verb methods have been rearranged because `condition` and `name` are optional.